resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -90,6 +90,14 @@ private[spark] object Config extends Logging {
       .toSequence
       .createWithDefault(Nil)
 
+  val CONFIG_MAP_MAXSIZE =
+    ConfigBuilder("spark.kubernetes.configMap.maxSize")
+      .doc("Max size limit for a config map. This is configurable as per" +
+        " https://etcd.io/docs/v3.4.0/dev-guide/limit/ on k8s server end.")
+      .version("3.2.0")
```

Comment:
It would be good, if this could make it into 3.1.0 ? @dongjoon-hyun and @srowen WDYT?

Reply:
On the one hand we normally only back-port fixes, and this is more like a better-error-message improvement. That said, for minor low-risk changes I am not against it if one feels at all strongly about it.